src/common/memory_tracking.hpp

```diff
@@ -253,10 +253,16 @@ enum {
     key_lnorm_tmp_diff_ss,
     key_lnorm_reduction,
     key_matmul_dst_in_acc_dt,
+    key_matmul_lt_algo_scratch,
+    key_matmul_lt_block_a,
+    key_matmul_lt_block_b,
```

Comment:
there already are keys for those (`key_gemm_blocked_a`, `key_gemm_blocked_b`). Any reason to add new ones?